tools/codegen/gen_backend_stubs.py

```diff
@@ -275,16 +318,10 @@ def gen_dispatcher_registrations(
                 class_method_name=f'{class_name}'),
             grouped_native_functions
         )),
-        'dispatch_registrations': list(concatMap(
-            dest.RegisterDispatchKey(
-                backend_index,
-                Target.REGISTRATION,
-                selector,
-                rocm=False,
-                cpp_namespace=cpp_namespace,
-                class_method_name=f'{class_name}'),
-            grouped_native_functions
-        )),
+        # If eager_registration is True, register the native functions
+        # for the dispatch key immediately on load. Otherwise, wait to
+        # register the functions until the generated registration func is explicitly invoked
+        'export_registration_func': "" if eager_registration else "TORCH_API",
```

Comment:
is this used? (this can probably just be folded into the `dispatch_registrations` param that the codegen either spits out a function or TORCH_LIBRARY block for)

Reply:
yea, this was leftover, i've removed it now.